src/vma/dev/rfs_uc.cpp

```diff
@@ -145,7 +145,6 @@ bool rfs_uc::prepare_flow_spec()
 bool rfs_uc::rx_dispatch_packet(mem_buf_desc_t* p_rx_wc_buf_desc, void* pv_fd_ready_array)
 {
 	// Dispatching: Notify new packet to the FIRST registered receiver ONLY
```

Comment:
Can we remove _reset_ref_count()_  call also from _rfs_uc_tcp_gro::add_packet_ ?